Add a line about HuggingChat in Code Llama blog post (#1423)
huggingface/blog: codellama.md

You can also check [this chat-based demo](https://huggingface.co/spaces/codellama/codellama-13b-chat) and clone it for your use – it's self-contained so you can examine the source code and adapt it as you wish!

> If you want to try out the bigger instruct-tuned 34B model, it is now available on **HuggingChat**! You can try it out here: [hf.co/chat](https://hf.co/chat)

Reviewer: ```suggestion
If you want to test the bigger instruct-tuned 34B model, it is now available on [**HuggingChat**](https://hf.co/chat)! Try it out in [hf.co/chat](https://hf.co/chat).
```

Reviewer: One more link, only if you think it's ok :)
